picard/util/__init__.py

```diff
@@ -71,6 +68,8 @@
     IS_WIN,
 )
 
+from dateutil.parser import parse
+
```

Comment:
You shouldn't move this import.

Reply:
As I've already mentioned, I couldn't commit without doing so.

https://pastebin.com/JhitSS5c

Reply:
That's isort again, that has a habit of changing between versions. Should install a version compatible with what is defined in https://github.com/metabrainz/picard/blob/master/requirements-dev.txt , so currently isort 4.3.10 or later, but not isort 5.